spec.md

```diff
@@ -379,26 +388,26 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id -d
 | Status Code  | Description  |
 |---|---|
 | 201 Created  | Service instance has been provisioned. The expected response body is below.  |
-| 200 OK  |  May be returned if the service instance already exists and the requested parameters are identical to the existing service instance. The expected response body is below. |
+| 200 OK  |  MAY be returned if the service instance already exists and the requested parameters are identical to the existing service instance. The expected response body is below. |
```

Comment:
I want to agree but I'm wondering if a 202 is a valid response as well in the cases where an async request is being replayed.  Sending back a 200 could impl the resource is ready, but in fact it might still be under construction.  Perhaps we could revisit this in a follow-on PR. I think it may just need a bit more language around it.